tensorboard/webapp/widgets/range_input/range_input_test.ts

```diff
@@ -271,6 +277,26 @@ describe('range input test', () => {
         });
       });
 
+      it('rounds to integer when returnIntegers is set to true', () => {
+        const {fixture, onRangeValuesChanged} = createComponent({
+          min: 0,
+          max: 6,
+          lowerValue: 1,
+          upperValue: 5,
+          tickCount: 10,
+          returnIntegers: true,
+        });
+        const [leftThumb] = getThumbsOnRange(fixture);
+        startMovingThumb(leftThumb);
+        moveThumb(leftThumb, 170);
+        // Without rounding, lowerValue would be 2.4.
+        expect(onRangeValuesChanged).toHaveBeenCalledWith({
+          lowerValue: 2,
+          upperValue: 5,
+          source: 'SLIDER',
+        });
```

Comment:
Can you add the test case when the lowerValue is 2.6 and round to 3?
Also lowerValue is 2 and the return is 2 just to be thorough?

Reply:
Added.